python/graphframes/graphframe.py

```diff
@@ -307,6 +309,34 @@ def pageRank(self, resetProbability = 0.15, sourceId = None, maxIter = None,
         jgf = builder.run()
         return _from_java_gf(jgf, self._sqlContext)
 
+    def parallelPersonalizedPageRank(self, resetProbability = 0.15, sourceIds = None,
+                                     maxIter = None):
+        """
+        Run the personalized PageRank algorithm on the graph,
+        from the provided list of sources in parallel for a fixed number of iterations.
+
+        See Scala documentation for more details.
+
+        :param resetProbability: Probability of resetting to a random vertex
+        :param sourceIds: the source vertices for a personalized PageRank
+        :param maxIter: the fixed number of iterations this algorithm runs
+        :return:  GraphFrame with new vertices column "pageranks" and new edges column "weight"
+        """
+        assert sourceIds is not None and len(sourceIds) > 0, "Source vertices Ids sourceIds must be provided"
+        assert maxIter is not None, "Max number of iterations maxIter must be provided"
+        if isinstance(sourceIds, list):
+            _gateway = self._sc._gateway
+            _srcIds = _gateway.new_array(_gateway.jvm.Object, len(sourceIds))
```

Comment:
We have to manually convert `Python list` to `Java Array[Any]`. Not sure if there is a better way.